pkg/operator/resource/resourceapply/rbac.go

```diff
@@ -80,7 +80,7 @@ func ApplyClusterRoleBinding(ctx context.Context, client rbacclientv1.ClusterRol
 
 	requiredCopy.RoleRef.APIGroup = rbacv1.GroupName
 	for i := range requiredCopy.Subjects {
-		if existingCopy.Subjects[i].Kind == "User" {
+		if requiredCopy.Subjects[i].Kind == "User" {
```

Comment:
Can you please fix also the same bug in `ApplyRoleBinding`?

Reply:
Oops, missed that, thanks. Done.